### PR TITLE
Adding secret module

### PR DESCRIPTION
**Pull request by andreaswittig**
Adding secret module as an alternative to define the password manually.

### CODE REVIEW

**Comment by andreaswittig** [comment]
test

File: README.md [hunk 1]
```diff
@@ -27,13 +27,14 @@ Resources:
...
         KmsKeyModule: '' # optional
+        SecretModule: '' # optional
         DBSnapshotIdentifier: '' # optional
         DBAllocatedStorage: '5' # optional
         DBInstanceClass: 'db.t2.micro' # optional
         DBName: '' # optional
         DBBackupRetentionPeriod: '30' # optional
         DBMasterUsername: 'master' # optional
-        DBMasterUserPassword: '' # required if DBSnapshotIdentifier is not set
+        DBMasterUserPassword: '' # required neither DBSnapshotIdentifier nor SecretModule is set
```
Comment (michaelwittig) [inline README.md]: if neither

**Review by michaelwittig — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (andreaswittig) [reply README.md]: Thanks, fixed!

**Comment by andreaswittig** [comment]
test

**Review by michaelwittig — APPROVED** [review]

**Comment by andreaswittig** [comment]
test

**Comment by andreaswittig** [comment]
test

**Pull request merged**